hawk/app/controllers/locations_controller.rb

```diff
@@ -149,6 +149,17 @@ def post_process_for!(record)
   end
 
   def normalize_params!(current)
+    if params[:location][:rules].nil?
+      params[:location][:rules] = []
+    else
+      params[:location][:rules] = params[:location][:rules].values
+    end
+
+    params[:location][:rules].each_with_index do |rule, index|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `rule`. If it's necessary, use `_` or `_rule` as an argument name to indicate that it won't be used.